### DISCUSSION TITLE
Can you change WebGL settings?

### DISCUSSION THREAD

**Discussion by bbkinglisbon** [ultrafunkamsterdam/undetected-chromedriver#1110] [Q&A]
I'm using this website to check some data. Is there any way to change WebGL Vendor and WebGL Renderer to

WebGL Vendor: Google Inc. (NVIDIA)
WebGL Renderer: ANGLE (NVIDIA, NVIDIA GeForce GTX 660 Direct3D11 vs_5_0 ps_5_0, D3D11)

Thanks for the help.

**Comment by PySCBist** [comment]
Try this:

    driver.execute_cdp_cmd('Page.addScriptToEvaluateOnNewDocument', {
        "source": "WebGLRenderingContext.prototype.getParameter = function(parameter) {if (parameter === 37445) {return 'Google Inc. (NVIDIA)';}if (parameter === 37446) {return 'ANGLE (NVIDIA, NVIDIA GeForce GTX 660 Direct3D11 vs_5_0 ps_5_0, D3D11)';}return getParameter(parameter);};"})

it's working for me, however, i still detected.

**Comment by niennguyen84** [comment]
I want to fake webgl vendor and webgl render,
Where i can get the list
Thank you

**Reply by dimaslanjaka** [reply]
same here, looking for the list
